### PR TITLE
Options for inline and label errors to remove duplicate messages and/or format them as a sentence.

### PR DESCRIPTION
**Pull request by PsiRadish**
Just pass a hash instead of a boolean to `inline_errors` and/or `label_errors`, containing your choice of `no_duplicates` and/or `to_sentence`.  Like so:

```
<%= bootstrap_form_for(@user, inline_errors: { no_duplicates: true, to_sentence: true }) do |f| %>
```

[Before](http://psiradish.github.io/img/rails-bootstrap-forms/before.png) and [After](http://psiradish.github.io/img/rails-bootstrap-forms/after.png) screenshot comparison.

### CODE REVIEW

**Comment by mattbrictson** [comment]
Thanks for the PR! I appreciate the screenshots. This looks like a good feature and I would like to merge it in after I get a release out the door for Rails 5.

In the meantime, I have a few suggestions for how to improve this PR:

* Add an example and a bit of documentation of this feature to the "Validation & Errors" section of the README.
* Include some basic unit tests.

Finally, the FormBuilder class is getting really huge, and I would like to avoid making it any bigger. Is there any way you can refactor your additions so that they reside in a separate helper object, maybe?

File: lib/bootstrap_form/form_builder.rb [hunk 1]
```diff
@@ -21,9 +21,9 @@ def initialize(object_name, object, template, options)
       @control_col = options[:control_col] || default_control_col
       @label_errors = options[:label_errors] || false
       @inline_errors = if options[:inline_errors].nil?
-        @label_errors != true
+        !@label_errors
```
Comment (koenpunt) [inline lib/bootstrap_form/form_builder.rb]: `@label_errors` isn't necessarily a boolean, so I think the comparison to a boolean is valid here.

**Comment by PsiRadish** [comment]
Unfortunately I haven't used Ruby or Rails since I made this PR, and even then it wasn't my strongest language/framework, so refactoring feels like a big ask.

For the same reason it might take some time (between work and Pillars of Eternity) to add unit tests, but a quick read of what's already in bootstrap_form_test.rb makes me think it's doable.

Adding documentation to the README should be no problem, excluding the aforementioned work-and-video-games time crunch.

**Comment by mattbrictson** [comment]
Thanks for the reply and for your honest assessment.

If you are still interested in doing the additional work for the README and the unit tests, then I will be happy to accept the PR without the additional refactoring. (Although please also address @koenpunt's comment about `!@label_errors`.)

However, if you are no longer really committed to Ruby and Rails, I totally understand if you'd rather let this one go. No worries.

**Comment by donv** [comment]
Closing since there has been no activity for 6 years.

**Pull request closed without merging**